wts/tokens.py

```diff
@@ -1,37 +1,73 @@
 import flask
-import requests
 import time
+import httpx
 
-from cdiserrors import AuthError, InternalError
+from cdiserrors import AuthError, InternalError, UserError
 
 from .models import db, RefreshToken
 from .utils import get_oauth_client
 
 
+def get_data_for_fence_request(refresh_token):
+    token = refresh_token.token
+    if hasattr(flask.current_app, "encryption_key"):
+        try:
+            token = str(
+                flask.current_app.encryption_key.decrypt(bytes(token, encoding="utf8")),
+                encoding="utf8",
+            )
+        except:
+            pass
```

Comment:
Do we just swallow if there happens to be an encryption key and we couldn't decrypt it?